src/bezpath.rs

```diff
@@ -118,7 +118,10 @@ impl BezPath {
     fn segments_of_slice<'a>(slice: &'a [PathEl]) -> BezPathSegs<'a> {
         let first = match slice.get(0) {
             Some(PathEl::MoveTo(ref p)) => *p,
-            Some(_) => panic!("First element has to be a PathEl::Moveto!"),
+            Some(PathEl::LineTo(ref p)) => *p,
```

Comment:
Incidentally, these `ref`s are no longer required, I think that changed with NLL.

Reply:
I guess it's the "match ergonomics" business. I didn't really follow that, tbh...